fenwick_tree/cpp/fenwick_tree.cpp

```diff
@@ -0,0 +1,54 @@
+#include <iostream>
+#include <vector>
+using namespace std;
+
+struct FenwickTree {
```

Comment:
Line ends in whitespace.  Consider deleting these extra spaces.  [whitespace/end_of_line] [4]

*Origin: CPPLintBear, Section: `all.cpplint`.*